.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - run: corepack enable
       - uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4.4.0
         with:
-          node-version: 18
+          node-version: 22
```

Comment:
should we bump to 20 instead, to catch any issues that might surface with 20, which is still supported?